pkg/util/http/http.go

```diff
@@ -0,0 +1,9 @@
+package util
```

Comment:
please add license header for a new file.

Reply:
modify http.go to util.go, and put common variables into util.go for introduction

Reply:
These have been modified